packages/costa/src/proto.ts

```diff
@@ -0,0 +1,23 @@
+export interface PluginMessage {
+  event: string;
+  params?: any[];
+}
+
+export enum PluginOperator {
+  START = 0x30,
+  WRITE = 0x31,
+  READ = 0x32,
+  COMPILE = 0x100
+}
+
+export class PluginProto {
+  op: PluginOperator;
+  message: PluginMessage;
+
+  static stringify(op: PluginOperator, message: PluginMessage): string {
+    return JSON.stringify({ op, message });
```

Comment:
Ditto?

Reply:
I dunno what do you mean, maybe in details?